### ISSUE TITLE
Docs: how does it compare to canopy

### ISSUE THREAD

**Issue by knocte** [JeremyBellows/marquee#3]
Would be great if in the first page of the documentation you could state how this project compares with canopy, why should people use it instead? Thanks

**Comment by voronoipotato** [comment]
Yes I agree. Saying  you were inspired by canopy begs the question "Why shouldn't we just use that". I'm sure you have some explanation as to what this does better or differently that you prefer. Even if it's just stylistic it would still be very useful.

**Comment by rowpsmo1859** [comment]
Thank you for the question!

It's great to hear feedback :+1: 

I'm going to update the docs to reflect the text below

------------

canopy is best used when 
- C# interop is required
- a quick and easy solution to UI Automation is needed to get up and running

marquee is best used when
- Team is comfortable with F#
- UI Automation is set as a long term strategy (marquee saves time and money by optimizing test execution and enables scalability through concurrent test design)

------------------
marquee Bottom Line
- Concurrent Test Execution - Save Time
- Scalable Test Suites - Save Money
- Intellisense for Browser interaction - Save Time (Especially with training)


--------------------

Thoughts?

**Comment by voronoipotato** [comment]
Ah! Yes that all makes a ton of sense. 
So to make it pithy 

Marquee is a F# First concurrent UI automation suite with intellisense.
